pkg/reconciler/ingress/config/store.go

```diff
@@ -28,7 +28,7 @@ import (
 type cfgKey struct{}
 
 // Config contains the configmaps requires for revision reconciliation.
-// +k8s:deepcopy-gen=false
+// +k8s:deepcopy-gen=true
```

Comment:
It looks to me that DeepCopy is being used.

https://github.com/knative-extensions/net-kourier/blob/dcf76e1d4f84d9f8427c86344b8c65ba173ef5e1/pkg/generator/ingress_translator_test.go#L971

Reply:
🤔 it might have been just generated once...